app/src/main/java/io/github/droidkaigi/confsched2017/view/helper/ResourceResolver.java

```diff
@@ -33,22 +35,20 @@ public String getString(@StringRes int resId, Object... formatArgs) {
 
     public String loadJSONFromAsset(final String jsonFileName) {
         String json = null;
-        InputStream is = null;
+        BufferedSource bufferedSource = null;
         try {
-            is = context.getAssets().open("json/" + jsonFileName);
-            int size = is.available();
-            byte[] buffer = new byte[size];
-            is.read(buffer);
-            json = new String(buffer, "UTF-8");
+            InputStream is = context.getAssets().open("json/" + jsonFileName);
+            bufferedSource = Okio.buffer(Okio.source(is));
+            json = bufferedSource.readUtf8();
         } catch (IOException e) {
-            Timber.e(TAG, e.getMessage(), e);
+            Timber.tag(TAG).e(e, "assets/json/%s: read failed", jsonFileName);
         } finally {
             try {
-                if (is != null) {
-                    is.close();
+                if (bufferedSource != null) {
+                    bufferedSource.close();
                 }
             } catch (IOException e) {
-                Timber.e(TAG, e.getMessage(), e);
+                // ignore
```

Comment:
Personally I think, it is commonly used that failure in finally block is ignored quietly.
(cf. [IOUtils.closeQuietly](https://commons.apache.org/proper/commons-io/javadocs/api-release/org/apache/commons/io/IOUtils.html#closeQuietly%28java.io.Closeable%29))

That said, it should be respected that project owners' opinion. 
Shall I put the log?